src/backend/cdb/cdbparquetfooterserializer.c

```diff
@@ -577,12 +620,14 @@ readRowGroupInfo(
 	if (!isset_columns)
 		ereport(ERROR,
 				(errcode(ERRCODE_GP_INTERNAL_ERROR), errmsg("file metadata row group column chunk not set")));
+	#if 0
 	if (!isset_total_byte_size)
 		ereport(ERROR,
 				(errcode(ERRCODE_GP_INTERNAL_ERROR), errmsg("file metadata row group total byte size not set")));
```

Comment:
Yes, these are some codes left due to the debugging process. Thanks for mentioning that. I will check that again.